ipaserver/dns_data_management.py

```diff
@@ -437,9 +453,11 @@ def update_base_records(self):
         success = []
         names_requiring_cname_templates = set(
             rec[0].derelativize(self.domain_abs) for rec in (
-                IPA_DEFAULT_MASTER_SRV_REC +
-                IPA_DEFAULT_ADTRUST_SRV_REC +
-                IPA_DEFAULT_NTP_SRV_REC
+                IPA_DEFAULT_MASTER_SRV_REC
+                + IPA_DEFAULT_MASTER_URI_REC
+                + IPA_DEFAULT_KRB_TXT_REC
+                + IPA_DEFAULT_ADTRUST_SRV_REC
+                + IPA_DEFAULT_NTP_SRV_REC
```

Comment:
Why change the position of the +?

Reply:
This was required by the linter in order to pass the checks.